server/src/main/java/org/elasticsearch/ingest/IngestStats.java

```diff
@@ -167,20 +170,28 @@ static Map<String, List<ProcessorStat>> merge(Map<String, List<ProcessorStat>> f
         return totalsPerPipelineProcessor;
     }
 
+    /**
+     * Read from a stream.
+     */
+    private static Stats readFrom(StreamInput in) throws IOException {
```

Comment:
Nit: You could rename this to `read` to match other cases that we use a static method to read from `StreamInput`, just like `IngestStats`.

Reply:
I cannot use read but I renamed to `readStats` which reflects better what the method is doing.